bitnami/airflow/templates/web/deployment.yaml

```diff
@@ -105,20 +105,28 @@ spec:
               value: {{ .Values.ldap.uri }}
```

Comment:
I think it is not needed to quote all the env. variables, in fact, the already existing ones are defined as a string in the _values.yaml_ and are not quoted here. We should follow the same approach for all of them

Reply:
I reviewed these values and I will remove those that are not needed (I can remove a couple of them I think). The ones that remain are needed to avoid being "evaluated" as YAML types (which raises the error below) so they are passed straight as strings to the container which will use them to prepare the webserver_config.yaml file.

Error:
`v1.EnvVar.Value: ReadString: expects " or n, but found t, error found in #10 byte of ...|,"value":true},{"nam|..., bigger context ...|{"name":"AIRFLOW_LDAP_USER_REGISTRATION","value":true},{"name":"AIRFLOW_LDAP_USER_REGISTRATION_ROLE"|...`

Reply:
Perfect, it's fine with that, just adding it when it is strictly necessary 👍

Reply:
thank you for your review it helped to improve it!